apps/prairielearn/src/models/enrollment.ts

```diff
@@ -664,7 +664,7 @@ export async function deleteEnrollment({
   return await runInTransactionAsync(async () => {
     const lockedEnrollment = await _selectAndLockEnrollment(enrollment.id);
 
-    assertEnrollmentStatus(lockedEnrollment, 'invited');
+    assertEnrollmentStatus(lockedEnrollment, ['invited', 'rejected']);
```

Comment:
I'm somewhat nervous about being able to delete enrollments, mostly because this gives people a means to bypass yearly institutional enrollment limits. If I wanted to do that at the end of a term, I would block all my students, re-invite them, and then immediately cancel all the invitations, which would delete the enrollments.

This seems bad.

There might be a bigger issue here w.r.t. how we could enrollments for yearly limits? That is, we might want to avoid counting unaccepted invites, rejections, blocks, etc. Idk exactly what to do about this.

Reply:
That is certainly a bug -- I don't think you can re-invite a blocked student. You can remove the block.

Reply:
Fixed in f952e0c47dc08507299d6c0f8f8b2d0e413ca016.